europe/README.EUROPE

```diff
@@ -294,3 +294,111 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
 Recommended to transform from Portuguese Datum 73 (EPSG:27493) to ETRS89 in Portugal (EPSG:3763).
 
 * D73_ETRS89_geo.gsb
+
+### Iceland: ISN93 -> ISN2016
+
+2D geodetic transformation from Iceland Datum ISN93 to Iceland Datum ISN2016
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+The transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+
+Recommended to transform from ISN93 2D geodetic (EPSG:4659) to ISN2016 2D geodetic (EPSG:8086), also this transformation has code in EPSG Dataset (EPSG:9232)
+
+* ISN93_ISN2016.gsb
+
+### Iceland: ISN2004 -> ISN2016
+
+2D geodetic transformation from Iceland Datum ISN2004 to Iceland Datum ISN2016
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+The transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+
+Recommended to transform from ISN2004 2D geodetic (EPSG:5324) to ISN2016 2D geodetic (EPSG:8086), also this transformation has code in EPSG Dataset (EPSG:9233)
+
+* ISN2004_ISN2016.gsb
+
+
+
+### Iceland: Deformation model: ISN93 -> ISN2016 
+
+*Source*: [IGN France](https://geodesie.ign.fr/index.php?page=grilles)   
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN93 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN93 3D geodetic (EPSG:4945) to ISN2016 3D geodetic (EPSG:8085)
```

Comment:
we could also probably add a custom entry in the PROJ database to register the PROJ pipeline between EPSG:4945 and EPSG:8085. A (really dummy) example of registering a PROJ pipeline string as a 
transformation is in https://github.com/OSGeo/PROJ/blob/master/test/unit/test_factory.cpp#L2074 . To be honest, this hasn't been used yet except in that unit test, so up to you to decide if you want attempt this or not :-)

Reply:
Yes, we would like to attempt it, here are the lines. Will you put the lines or should I?
EPSG:4945 To EPSG:8085
+proj=pipeline   +step +init=ISL:ISN93_16h   +step +inv +init=ISL:ISN93_16v 
expending will be
 +proj=pipeline   +proj=hgridshift +grids=ISN93_ISN2016.gsb  +step +inv +proj=vgridshift +grids=ISN93_ISN2016.gtx vunits=m 

EPSG:5323 To EPSG:8085
+proj=pipeline   +step +init=ISL:ISN04_16h   +step +inv +init=ISL:ISN04_16v

expending will be
 +proj=pipeline   +proj=hgridshift +grids=ISN2004_ISN2016.gsb  +step +inv +proj=vgridshift +grids=ISN2004_ISN2016.gtx vunits=m

Reply:
I'm not sure I'll have the time myself to tackle that. I would strongly suggest using the expended syntax. And perhaps consider this as a follow up action after your initial submission. As I said, I'd consider this as an experimental feature as it has been little tested, so code changes might potentially be also required.

Reply:
We will skip this approach.